src/Security/ResourceOwner/Doctrine/DBALResourceOwnerRepository.php

```diff
@@ -63,10 +63,27 @@ public function markResourceEditableByUser(string $resourceId, string $userId):
 
     public function markResourceEditableByNewUser(string $resourceId, string $userId): void
     {
-        $this->connection->update(
-            $this->tableName,
-            ['user_id' => $userId],
-            [$this->idField => $resourceId]
+        $results = $this->connection->fetchAll(
+            'SELECT * FROM ' . $this->tableName . ' WHERE event_id = :eventId',
```

Comment:
Maybe we better stick with the DBAL query language instead of writing SQL statements. For example on line 33.